core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -215,6 +215,12 @@ oppia.controller('CollectionPlayer', [
       return completedExplorationIds.indexOf(explorationId) > -1;
     };
 
+    $scope.isNextExploration = function(explorationId) {
+      var nextExplorationId = (
+        $scope.collectionPlaythrough.getNextExplorationIds()[0]);
+      return (nextExplorationId === explorationId) ? true : false;
```

Comment:
Ah ok, next time please remember to mark it as "done" (or "obsolete with the new changes") so that it's clear it's been addressed. Thanks!